### ISSUE TITLE
ML Model Implementation

### ISSUE THREAD

**Issue by VasLem** [MDAIceland/WaterSecurity#31]
The next step to take after #10 . There is an idea of the dimensionality of features/labels. So that shall be enough to create a model prototype, based on the research. The model can be created by making use of [pipelines](https://scikit-learn.org/stable/modules/generated/sklearn.pipeline.Pipeline.html) and [subclassing base classes](https://scikit-learn.org/stable/developers/develop.html#rolling-your-own-estimator), to package the model into an sklearn-like class.

**Comment by ekaan** [comment]
@antosalerno do you need any assistance or you are just changing the parameters and see which one will work the best? I might try to help

**Comment by antosalerno** [comment]
Exactly, that's what I'm doing. I'm performing cross validation and storing in a file the best parameters.

**Comment by VasLem** [comment]
Could you please communicate the structure of the expected file, so that I can start working on the python file equivalent implementation? @antosalerno

**Comment by antosalerno** [comment]
Is it ok a dictionary saved as .pkl file?

**Comment by antosalerno** [comment]
Great @VasLem! 
Did you use BayesianOptimization as you were suggesting in the call?
Can you retrieve the iteration minimising the loss function for each risk?

**Comment by antosalerno** [comment]
Great @VasLem!

**Comment by VasLem** [comment]
This is not the final version, I just plugged things in, if anyone of you had managed to get better results, please do your commits!!!

**Comment by VasLem** [comment]
@OlympiaG do we have any news about the Bagging option? Currently I report mse close to 1 for most risks, which sucks a little...
